### PR TITLE
Delete expired user tokens with maintenance cleanup command

### PR DESCRIPTION
**Pull request by hluchas**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
There is no feature which deletes expired user tokens.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
None.

#### Steps to test this PR:
1. You have to create some record in `user_tokens` table with past and future expiration date.
2. `app/console mautic:maintenance:cleanup --dry-run`
3. `app/console mautic:maintenance:cleanup` 
4. Check table content

#### List deprecations along with the new alternative:
None.

#### List backwards compatibility breaks:
None.

### CODE REVIEW

**Comment by mleffler** [comment]
This has had one test pass and has been code reviewed.

File: app/bundles/CoreBundle/Test/EventListener/MaintenanceSubscriberTest.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Test\EventListener;
+
+class MaintenanceSubscriberTest extends \PHPUnit_Framework_TestCase
```
Comment (Woeler) [inline app/bundles/CoreBundle/Test/EventListener/MaintenanceSubscriberTest.php]: What's this class for?

Comment (hluchas) [reply app/bundles/CoreBundle/Test/EventListener/MaintenanceSubscriberTest.php]: For testing, but doesn't contain any code :-)
I'll push fix.

Comment (hluchas) [reply app/bundles/CoreBundle/Test/EventListener/MaintenanceSubscriberTest.php]: Fixed

**Comment by hluchas** [comment]
@Woeler
Do you have an idea why PHPUnit fails on PHP7.0?

I'm using https://github.com/dg/bypass-finals. It seems that this concept or exactly this library has problem with PHP7.0 version.

**Comment by Woeler** [comment]
@hluchas Yea, it's a problem between Travis and Maxmind (ip lookup). It has nothing to do with your code. I have messaged them, but it seems to be working again on other PR's, so I re-triggered your build.

**Comment by hluchas** [comment]
@Woeler Updated UserTokenRepositoryInterface, dependent MaintenanceSubscriber and removed dg/bypass-finals package as not needed now.

**Comment by hluchas** [comment]
@Woeler Failed on PHP7.1 with the same errors. Pls re-trigger build.

**Comment by Woeler** [comment]
@hluchas It won't matter. I have contacted Maxmind and they need to solve it on their end. Those tests won't pass until they add Travis to their whitelist. We need to wait for their reply.

**Pull request merged**